client/js/components/user/DpToggleForm.vue

```diff
@@ -0,0 +1,62 @@
+<template>
```

Comment:
I suggest to drop `Dp` from the component name, as it is an app component. 

[Our docs are not up to date](https://dplan-documentation.demos-europe.eu/development/guidelines-conventions/coding-styleguides/javascript.html#folder-structure-prefixing-and-naming-of-components) in this point, however `Dp` prefix should only be used in demosplan-ui.

Reply:
Done in https://github.com/demos-europe/demosplan-core/pull/2026/commits/e0de436c884ba651681c512ea82b4e973e895701